sass/components/_gameMode.scss

```diff
@@ -0,0 +1,14 @@
+.gameMode {
+  height: 10rem;
```

Comment:
Ta linijka jest dla mnie abstrakcją. Nie rozumiem jej.
Mamy uzależniać wielkość przycisku od czcionki? o.0
Bo rem/em odnosi się tylko czcionki, zgadza się?

Reply:
Czy to nie jest proszenie się o nieszczęście? 
(mogę nie mieć tu wiedzy, więc proszę o wytłumaczenie)

Reply:
Czy to nie jest proszenie się o nieszczęście?
(mogę nie mieć tu wiedzy, więc proszę o wytłumaczenie)

Reply:
Może ja też nie mam tutaj super wiedzy, ale jak dla mnie to jest OK. Skoro przycisk ma napis, no to wielkość fonta wpływa na wielkość przycisku. Też bym tak zrobił :) Jakie kłopoty masz na myśli?